src/index.html

```diff
@@ -16,11 +16,31 @@
 </head>
 
 <body>
+  <div class="header">
+  <h1><a class="name" href="https://github.com/Stephenich/stephenich.github.io">Stehen Gurney</a></h1>
+    <img src="images/githubIcon.svg" alt="GitHub Logo">
+  </div>
+   <header> <div class="menu">
+      <ul>
+        <li>Home</li>
+        <li>GitHub</li>
+      </ul>
+    </div>
+  </header>
```

Comment:
a better semantic layout would be something like this 

```
<header>
    <div className="header">
        <h1>
            <a
                className="name"
                href="https://github.com/Stephenich/stephenich.github.io">
                Stehen Gurney
            </a>
        </h1>
        <img
            src="images/githubIcon.svg"
            alt="GitHub Logo"
        />
    </div>
    <nav className="menu">
        <ul>
            <li>Home</li>
            <li>GitHub</li>
        </ul>
    </nav>
</header>
```

Reply:
I'll be sure to change it to be more semantic and readable.